docs/aurora/running-jobs-aurora.md

```diff
@@ -10,6 +10,9 @@ There is a single routing queue in place called `EarlyAppAccess` which submits t
 - `lustre_scaling` (execution queue) : 10 running jobs per-user; max walltime : 6 hours; max nodecount : 9090 (subject to change)
 
 ### Submitting a job
+
+Note: Jobs should be submitted only from your allocated project directory and not from your home directory. 
+
```

Comment:
Not so clear about what you want to say here. 

It is perfectly fine to submit jobs from the home directory. It just the data is suggested to be in Lustre or DAOS, right?

Reply:
removed.